db/seeds.rb

```diff
@@ -29,11 +29,11 @@
 Ingredient.create(name:'Chocolate Malt', sub_type: 'Grain',
  extract_potential: 1.028, alpha_acid: nil, attenuation: nil, main_class: "Grain")
 Ingredient.create(name:'Melanoiden Malt', sub_type: 'Grain',
- extract_potential: 1.037, alpha_acid: nil, attenuation: nil , main_class: "Grain")
+ extract_potential: 1.037, alpha_acid: nil, attenuation: nil, main_class: "Grain" )
```

Comment:
Line is too long. [83/80]<br>Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.